### ISSUE TITLE
Many images have just one bounding box for a class that in reality has multiple instances on that image

### ISSUE THREAD

**Issue by lamerman** [openimages/dataset#65]
For example this image:

![download](https://user-images.githubusercontent.com/1468837/36303738-72dd5280-130d-11e8-9ead-a06f1b6cfefa.png)

It has multiple chairs, but only one of them has bounding box.

Is it how it was classified or was it done intentionally?

**Comment by rkrasin** [comment]
@samihaija Sami, can you please comment on this?

**Comment by samihaija** [comment]
Ivan, thank you for the mention!

The above is intentional, assuming the image came from the training partition. We only have one box per image per entity, for the training set. However, for the validation set, we tried to have all instances boxed.

**Comment by lamerman** [comment]
@samihaija @rkrasin  thank you.

@samihaija I'm not a big expert in object detection algorithms, but my first guess would be that a neural network that is learning on this data will be penalized while training. It predicts multiple chairs, but the training data has only one and my guess would be that when it predicts YES and the training data says it NO, when in reality it's YES, the network will be penalized for such predictions. 

It much more a question than statement, as I am not sure.

What do you think, could be a problem?

**Comment by lamerman** [comment]
I looked at the loss function of YOLO 

![screenshot_20180220_181624](https://user-images.githubusercontent.com/1468837/36438447-415676a0-166a-11e8-9bea-aa6b3d9f58b3.png)

And it seems like absence of bounding box for image when in reality it should be will affect the loss function on line 4. And it's interesting what was the reasoning behind making only one bounding box for openimages.

**Comment by phdung** [comment]
I am curious about this method also. @lamerman @samihaija any updates about this?

**Comment by dbrazey** [comment]
Does someone has an explanation concerning the fact that training data contains only one bounding box instead of all boxes ?
 
As a result, this dataset cannot be used to train object detection algorithm ?

**Comment by dashesy** [comment]
because of this, Yolo does not train well with this dataset.

**Comment by ys707221924** [comment]
@dashesy hi, do you try any other detection algorithm ( like faster rcnn, ssd ) with openimages dataset? I try to train faster rcnn with mxnet with openimages, but I have many problems when preprocessing the dataset.

**Comment by ys707221924** [comment]
@dashesy hi, do you try any other detection algorithm ( like faster rcnn, ssd ) with openimages dataset? I try to train faster rcnn with mxnet with openimages, but I have many problems when preprocessing the dataset.

**Comment by kevinsu628** [comment]
Further, a lot of images have missing labels. Is this problem fixable? 


![openImage_issue_1](https://user-images.githubusercontent.com/19247030/66378623-6c493e00-e9e6-11e9-946f-badca4124a82.png)
![openImage_issue_2](https://user-images.githubusercontent.com/19247030/66378627-6e130180-e9e6-11e9-902b-15b2cd734ea9.png)
